ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-x86.yaml

```diff
@@ -115,8 +116,8 @@ tests:
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
       COMPUTE_NODE_REPLICAS: "24"
-      COMPUTE_NODE_TYPE: Standard_D4s_v3
-      CONTROL_PLANE_INSTANCE_TYPE: Standard_D4s_v3
+      COMPUTE_NODE_TYPE: Standard_D4s_v5
+      CONTROL_PLANE_INSTANCE_TYPE: Standard_D4s_v5
```

Comment:
Please update the Control Plane Instance Type: `Standard_D8s_v5`

Reply:
I did the update.
But attention is needed here for some inconsistent setting: for the control plane instance type, I referenced to AWS [config](https://github.com/openshift/release/blob/master/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.17-nightly-x86.yaml#L56-L67), it is using SIZE_VARIANT: default(4 vCPU, 16GB control plane nodes, default workers. . m6a. or m6i.). But according to https://docs.google.com/document/d/1sVkbU5ttS3g_GqfEeAWDzT25asvy_jKqy0qGCJ4Lfzo/edit#heading=h.bqtllo1nrbil, small scale 24 nodes 8vCPU instance.

Reply:
Changed back to Standard_D4s_v5 to be same as the AWS config for control-plane-24nodes